src/mudlet.cpp

```diff
@@ -2058,9 +2053,10 @@ void mudlet::slot_show_connection_dialog()
     auto pDlg = new dlgConnectionProfiles(this);
     connect(pDlg, SIGNAL(signal_establish_connection(QString, int)), this, SLOT(slot_connection_dlg_finished(QString, int)));
     pDlg->fillout_form();
-    if (pDlg->exec() == QDialog::Accepted) {
```

Comment:
If the connection dialog was brought up in the past, because it was activated by the `exec()` call it would block the rest of the application - which is _bad_ if you are multi-playing as using it to open a second or more profile would block the existing ones! Using `show()` is better but I am not yet certain that the rest of the class/application will behave in the same way should the end of `(void) mudlet::slot_show_connection_dialog()` be reached before a connection is _started to be made_ at least - is the sequencing of the various signals and slots still good?